base_product_merge/tests/test_base_product_merge.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
```suggestion
# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Reply:
The licence changed to AGPL because of openupgrade merge tool